pkg/config/validation/validation.go

```diff
@@ -185,6 +185,24 @@ func validateDNS1123Labels(domain string) error {
 	return nil
 }
 
+// validate the trust domain format
+func ValidateTrustDomain(domain string) error {
+	if len(domain) == 0 {
+		return fmt.Errorf("empty domain name not allowed")
+	}
+	parts := strings.Split(domain, ".")
```

Comment:
this seems to be the same as validateDNS1123Labels, can we just use that?

Reply:
Thanks for reviewing. There are couple differences, the trust domain is not limited with max length or pure number, plus the error message. Please let me know if any of them not valid, I can update them accordingly.